composer.json

```diff
@@ -28,13 +29,17 @@
         "symfony/yaml": "^4.1"
     },
     "require-dev": {
+        "csa/guzzle-cache-middleware": "^1.0",
```

Comment:
weird that this is actually used for mocking, rather than for caching

Reply:
but `require-dev` is the right place then

Reply:
It's annoying actually a hard dependency of `csa/guzzle-bundle`, but we only use it here.